js/task_with_timeout.js

```diff
@@ -0,0 +1,50 @@
+/*
+ * vim: ts=4:sw=4:expandtab
+ */
+(function () {
+    window.Whisper = window.Whisper || {};
+
+    Whisper.createTaskWithTimeout = function(task, id) {
```

Comment:
Also, this timeout stuff is now in this PR over here: https://github.com/WhisperSystems/Signal-Desktop/pull/1317